### PR TITLE
Add getting started guide for Data Tiering

### PR DESCRIPTION
**Pull request by gayyappan**
# Description

Add getting started guide for Data tiering

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/data-tiering/querying-tiered-data.md [hunk 1]
```diff
@@ -11,11 +11,13 @@ tags: [ storage, data management ]
 <!-- vale Google.Acronyms = NO -->
 <!-- vale Google.Headings = NO -->
 
-Once rarely used data is tiered and migrated to low cost object storage it can still be queried with standard SQL by
-enabling the `timescaledb.enable_tiered_reads` GUC. 
+Once rarely used data is tiered and migrated to low cost object storage it can still be queried 
+with standard SQL by enabling the `timescaledb.enable_tiered_reads` GUC.
```
Comment (iroussos) [inline use-timescale/data-tiering/querying-tiered-data.md:15]: I think that we should add a note that querying is turned off by default. What's your opinion?

**Review by iroussos — APPROVED** [review]
Thank you @gayyappan, looks good to me - I just added a very minor comment for your concideration.

Let's merge this and follow-up with the rest of the updates we want to make.

File: use-timescale/data-tiering/querying-tiered-data.md (hunk 1 above)
Comment (gayyappan) [reply use-timescale/data-tiering/querying-tiered-data.md:15]: Agree. I am not sure if that change has been deployed yet. Need to verify before updating the doc.

File: use-timescale/data-tiering/tour-data-tiering.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+Consider a simple database with a standard devices table and a metrics hypertable.
+
+```
+CREATE TABLE devices ( id integer, description text);
+CREATE TABLE metrics ( ts timestamp with time zone, device_id integer, val float);
+SELECT create_hypertable('metrics', 'ts');
+```
+
+Once you insert data into the tables, you can then tier some of the hypertable’s data to object storage.  A simple query against the nformational view illustrates which chunks are tiered to object storage.
```
Comment (matthewmcnew) [inline use-timescale/data-tiering/tour-data-tiering.md:147]:
```suggestion
Once you insert data into the tables, you can then tier some of the hypertable’s data to object storage.  A simple query against the informational view illustrates which chunks are tiered to object storage.
```

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**